### ISSUE TITLE
Build error with v3.3.1

### ISSUE THREAD

**Issue by wizd** [vercel/ai#2569]
### Description

after upgrading ai to v3.3.1 and @ai-sdk/openai to 0.0.41, my next.js project gets build error:

```
Failed to compile.

./node_modules/.pnpm/ai@3.3.1_openai@4.52.7_react@18.3.1_solid-js@1.8.15_sswr@2.1.0_svelte@4.2.12__svelte@4.2.12_v_wlqrfbcon3levo6qbcnfgtjvz4/node_modules/ai/dist/index.mjs
Attempted import error: 'AISDKError' is not exported from '@ai-sdk/provider' (imported as 'AISDKError').

Import trace for requested module:
./node_modules/.pnpm/ai@3.3.1_openai@4.52.7_react@18.3.1_solid-js@1.8.15_sswr@2.1.0_svelte@4.2.12__svelte@4.2.12_v_wlqrfbcon3levo6qbcnfgtjvz4/node_modules/ai/dist/index.mjs
./app/api/chat/route.ts
```

**Comment by lgrammel** [comment]
@wizd do you import other `@ai-sdk` packages?

**Comment by wizd** [comment]
yep. I did rm -rf node_modules and .next folder and build again. the same error.

**Comment by lgrammel** [comment]
Can you check your `@ai-sdk/provider` version? 

it should be `0.0.14`, which `@ai-sdk/openai` uses. That version exports `AISDKError`.

**Comment by wizd** [comment]
adding this package has no help...
```
pnpm add @ai-sdk/provider

dependencies:
+ @ai-sdk/provider 0.0.14
```

**Comment by lgrammel** [comment]
well you should already have it. if you use pnpm, do you have a repository to share that i can look at? it's probably just some information not fully updated

**Comment by wizd** [comment]
```
grep -ir AISDKError ./node_modules/@ai-sdk/provider
```

found nothing, strange?

**Comment by lgrammel** [comment]
You're correct, it was not released for some reason. Sigh.

**Comment by OrhanTozan** [comment]
I have the same error.

**Comment by lgrammel** [comment]
I'll try to get a new release ready. Something must have gone wrong in the previous release.

**Comment by OrhanTozan** [comment]
@lgrammel how long will it take? Which version can I use in the meantime?

**Comment by lgrammel** [comment]
Maybe 10-15 minutes. please use the previous version for now (1 patch version lower)

**Comment by lgrammel** [comment]
Please try `ai@3.3.2` and `@ai-sdk/openai@0.42`

**Comment by wizd** [comment]
upgraded and build successfully. thank you! 

> Please try `ai@3.3.2` and `@ai-sdk/openai@0.0.42`

**Comment by lgrammel** [comment]
## Identify issue type

Identified as **Bug**  
Confidence: **High**

### Rationale
This was a confirmed packaging/release regression that broke builds and was fixed in subsequent patch releases.

### Evidence
- Upgrading to ai@3.3.1 and @ai-sdk/openai@0.0.41 caused a Next.js build failure because AISDKError was missing from @ai-sdk/provider.
- The maintainer confirmed the required export was not released and said the previous release had gone wrong.
- The corrected releases ai@3.3.2 and @ai-sdk/openai@0.0.42 resolved the build for affected users.
